lib/index.js

```diff
@@ -450,6 +451,15 @@ class RustLanguageClient extends AutoLanguageClient {
     params.initializationOptions = params.initializationOptions || {}
     // Don't build straight after initialize, wait for first `workspace/didChangeConfiguration`
     params.initializationOptions.omitInitBuild = true
+
+    try {
+        let rlsConfigPath = path.join(projectPath, "rust-analyzer.json")
+        let options = fs.readFileSync(rlsConfigPath)
```

Comment:
Could you first check for the existence of the file using `existsSync`?

Reply:
Sure, just added the commit